src/intro/useCases/utils/__tets__/utils.test.ts

```diff
@@ -0,0 +1,16 @@
+import { describe, expect, it } from 'vitest';
+import { getGreeting } from '../getGreeting';
+
+describe('utils', () => {
+    describe('getGreeting', () => {
+        it("should return 'Good morning' when hour is less than 12", () => {
```

Comment:
A good idea is to maximally decouple test description from its details. For instance, when a new business rule comes in to change the "Good morning" to "Hey", we want to maintain only the assertion behind the test case. Not its description:

```suggestion
        it("should return greeting for morning hours", () => {
```

(ditto throughout)